dns/ddclient/src/opnsense/service/templates/OPNsense/ddclient/ddclient.conf

```diff
@@ -53,6 +53,8 @@ zone={{account.zone}}, \
 {%      elif account.service == 'dns-o-matic' %}
 protocol=dyndns2, \
 server=updates.dnsomatic.com, \
+{%      elif account.service == 'domeneshop' %}
+protocol=domeneshop, \
```

Comment:
when it's only available in the native service, you shouldn't need a ddclient config.

Reply:
OK, then I misunderstood the ddclient.conf.